plugins/modules/rabbitmq_plugin.py

```diff
@@ -66,6 +75,12 @@
     names: rabbitmq_management,rabbitmq_management_visualiser,rabbitmq_shovel,rabbitmq_shovel_management
     state: enabled
     new_only: 'yes'
+
+- name: Enables the rabbitmq_peer_discovery_aws plugin without requiring a broker connection.
```

Comment:
It looks like it's enabling the ``rabbitmq_management`` plugin, not the ``rabbitmq_peer_discovery_aws`` plugin.

Reply:
Oh, 😞 that's embarrassing haha. Just fixed it. Thank you for the prompt review!